### ISSUE TITLE
prefer-const with exported 'let'

### ISSUE THREAD

**Issue by bookman25** [Microsoft/tslint-microsoft-contrib#322]
TypeScript 2.1.4

Exporting a variable is triggering this rule with the latest version of TypeScript.  It was working correctly with TS 2.0

```
export let value: string;
```

**Comment by HamletDRC** [comment]
thanks, i'll take a look.

**Comment by bookman25** [comment]
Any updates?

**Comment by bookman25** [comment]
Looks like this is covered by core tslint now: https://github.com/palantir/tslint/pull/1801

**Comment by HamletDRC** [comment]
This is on the #releases branch now in case you need to point your npm to that: 

    npm install git://github.com/Microsoft/tslint-microsoft-contrib.git#releases
